packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -240,8 +241,11 @@ class Checkbox extends StatefulWidget {
   /// {@macro flutter.widgets.Focus.autofocus}
   final bool autofocus;
 
-  /// The Border Radius of the checkbox.
-  final double borderRadius;
+  /// The Border of the checkbox.
```

Comment:
Please don't just reiterate the property name as documentation. Explain what the shape contributes to the checkbox visuals. 

If you refer to a significant type then use `[Foo]`.

If we're going to allow developers to specify the checkbox's shape, then the type of this property should be generic. In this case I'd suggest [OutlinedBorder](https://api.flutter.dev/flutter/painting/OutlinedBorder-class.html)

Please explain what this property's default value is in terms of this parameter, and the theme.